htdocs/fourn/facture/paiement.php

```diff
@@ -555,14 +555,12 @@ function callForResult(imgId)
 							print '<td class="right">'.$langs->trans('MulticurrencyAmountTTC').'</td>';
 							print '<td class="right">'.$langs->trans('MulticurrencyAlreadyPaid').'</td>';
 							print '<td class="right">'.$langs->trans('MulticurrencyRemainderToPay').'</td>';
+							print '<td class="center">'.$langs->trans('MulticurrencyPaymentAmount').'</td>';
```

Comment:
You should use add here
`if (!empty($conf->multicurrency->enabled)) {`
If module is off, we should not get any feature relative to multicurrency